cmd/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+// current implementation is for sqlite
+import (
+	"net/http"
+
+	gin "github.com/gin-gonic/gin"
+
+	sqlite_controller "github.com/leechongyan/Studtor_backend/internal/controllers/sqlitecontroller"
+)
+
+func main() {
+	// Set the router as the default one shipped with Gin
+	router := gin.Default()
+
+	// Serve 404 page when page not found
```

Comment:
Even though it's convenient to place routes here, I don't think that should be the case. ```Main``` should only be concerned with setting up app requirements (eg: maybe logging on prod directs to cloudwatch).

Would it be better to instead have a folder for the routes or to group the files by their main concern (here students) then import the routes from there?

Reply:
Addressed by @leechongyan , PR #18